contracts/helpers/Blacklist.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.23;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract Blacklist is Ownable {
+    mapping(address => bool) public blacklisted;
+
+    constructor(address owner) Ownable(owner) {}
+
+    event BlacklistAddress(address pool, bool isBlacklisted);
```

Comment:
This form isn't in line with the naming of the rest events in the project. What do you think about `BlacklistUpdated`?